packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.types.ts

```diff
@@ -149,6 +164,21 @@ export interface IColorPickerGridCellStyleProps {
    * Whether the color being rendered is white or not. If it is white we show a border around it.
    */
   isWhite?: boolean;
+
+  /**
+   * The height of this cell, in pixels.
+   */
+  height?: number;
```

Comment:
Add `@default` values for documentation please for all the new props which have a default value.